lib/commands/project/project-command-base.ts

```diff
@@ -0,0 +1,12 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+export class ProjectCommandBase {
```

Comment:
Actually, this check must be in `canExecute`. I realize it after the failed QA tests :smile_cat: